### PR TITLE
Fixing french translation

### PR DESCRIPTION
**Pull request by BoFFire**
Updating arrays in french.

### CODE REVIEW

**Comment by Swati4star** [comment]
Can you merge these 2 PRs into single one?
https://github.com/Swati4star/Images-to-PDF/pull/785

Make the changes in a single branch only

**Comment by BoFFire** [comment]
> Can you merge these 2 PRs into single one?
> #785
> 
> Make the changes in a single branch only

Hi, sorry I'm not really a Github user, I don't know how things work here. I only translate FLOSS.
Regards,

**Comment by BoFFire** [comment]
Is it okay ? ^^

File: app/src/main/res/values-fr/strings.xml [hunk 1]
```diff
@@ -2,425 +2,302 @@
...
+    <string name="excel_selected">Fichier Excel sélectionné</string>
+    <string name="excel_to_pdf">Excel vers PDF</string>
+    <string name="excel_tv_view_text">Fichier Excel sélectionné :</string>
+    <string name="select_excel_file">Sélectionnez un fichier Excel</string>
+    <string name="add_to_favourite">Ajouter aux favoris</string>
+    <string name="favourites">Favoris</string>
+    <string name="favourites_text">Sélectionnez l\'icône + pour ajouter vos fonctionnalités préférées !</string>
+    <string name="split_one_page_pdf_alert">Le PDF sélectionné ne peut pas être divisé car il ne contient qu\'une seule page !</string>
     <string name="no_excel_file">Aucun fichier Excel sélectionné</string>
-    <string name="split_range_alert">Entered split range will create Selected PDF again!</string>
```
Comment (Swati4star) [inline app/src/main/res/values-fr/strings.xml]: You have removed some of the strings @BoFFire 
For example, this one `split_range_alert`?

File: app/src/main/res/values-fr/strings.xml [hunk 2]
```diff
@@ -2,425 +2,302 @@
...
-        <item>Add Watermark</item>
+        <item>Ajouter une filigrane</item>
     </array>
-
-    <string name="please_wait">Patientez</string>
+    <string name="please_wait">Veuillez patienter</string>
     <string name="create_pdf">Accueil</string>
-    <string name="viewFiles">Voir Fichiers</string>
-
-    <string name="bundleKey">ImageUri</string>
```
Comment (Swati4star) [inline app/src/main/res/values-fr/strings.xml]: this too

File: app/src/main/res/values-fr/strings.xml (hunk 1 above)
Comment (BoFFire) [reply app/src/main/res/values-fr/strings.xml]: Solved.

File: app/src/main/res/values-fr/strings.xml (hunk 2 above)
Comment (BoFFire) [reply app/src/main/res/values-fr/strings.xml]: Solved. New file submitted.

**Comment by BoFFire** [comment]
Generally, on Android when a string doesn't exist or is missing on values-xx it fallsback automatically to `values` strings in english.
Regards,

File: app/src/main/res/values-fr/strings.xml [hunk 3]
```diff
@@ -1,426 +1,399 @@
...
-    <string name="whatsappToast">Sorry! Images could not be added. Please select the images from our **Select Images** option.</string>
```
Comment (Swati4star) [inline app/src/main/res/values-fr/strings.xml]: this is still deleted

Comment (BoFFire) [reply app/src/main/res/values-fr/strings.xml]: Hi, I put back all the untranslated strings in english which will be translated later :)
Regards,

**Comment by Swati4star** [comment]
I'll merge this.
But a suggestion for next time, don't change the order of strings too. It becomes difficult to review what changes you made in "Files changed" section

**Pull request merged**